src/quo2/components/lowest_price.cljs

```diff
@@ -0,0 +1,49 @@
+(ns quo2.components.lowest-price
+  (:require [quo.design-system.colors :as colors]
+            [quo.react-native :as rn]))
+
+(defn line-styles [flex-grow]
+  {:opacity 0.4
+   :border-style :dotted
+   :border-width 1
+   :border-color (:text-01 @colors/theme)
+   :flex-grow flex-grow
+   :overflow :hidden
+   :border-radius 1})
+
+(def centrify {:flex-direction :row
+               :justify-content :center
+               :align-items :center})
+
+(defn border-alignment [color]
+  {:align-self :flex-start
+   :padding-horizontal 2
+   :padding-vertical 2
+   :flex-grow 0.25
+   :border-radius 3
+   :background-color color})
+
+(defn lowest-price
+  [{:keys [top-value
+           bottom-value
+           margin-top
+           top-value-text-color
+           top-value-bg-color
+           bottom-value-bg-color
+           bottom-value-text-color]}]
+  [rn/view {:style {:flex-direction :column
+                    :width "100%"
+                    :margin-top (int margin-top)
+                    :justify-content :center}}
+   [rn/view {:style centrify}
+    [rn/view {:style (line-styles 1)}]
+    [rn/view {:style (border-alignment top-value-bg-color)}
+     [rn/text {:style {:color top-value-text-color
+                       :text-align :center}} top-value]]
+    [rn/view {:style (line-styles 4)}]]
+   [rn/view {:style centrify}
+    [rn/view {:style (line-styles 1)}]
+    [rn/view {:style (border-alignment bottom-value-bg-color)}
+     [rn/text {:style {:color bottom-value-text-color
+                       :text-align :center}} bottom-value]]
+    [rn/view {:style (line-styles 1)}]]])
```

Comment:
I am not sure if we are going to use this component as it is. Better to discuss with design team but probably only one of lowest price component will be shown as per position in graph. Something like this example
<img src="https://user-images.githubusercontent.com/17097240/184991278-9a61f3c9-16c6-4c0a-a2a0-7d06b086c3fd.png" alt="Screenshot" height="100"/>
<img src="https://user-images.githubusercontent.com/17097240/184992443-561a0e7c-2181-4415-aac8-7115e89da6a7.png" alt="Screenshot" height="200"/>

Reply:
Will try to refactor it to use less views